### ISSUE TITLE
Cant Send from TezBox to BakeChain

### ISSUE THREAD

**Issue by sbusher78** [tezbox/desktop-wallet#6]
I am trying to send funds OUT of my Tezbox wallet to BakeChain and I keep getting a failure and it tells me to check my inputs.  I've restored the wallet onto a different PC using the private keys and the issue persists.  I also cant update the delegate for some reason.  Thanks in advance for your help

**Comment by stephenandrews** [comment]
Please provide addresses and details about the failed transaction, fees, settings you are using

**Comment by sbusher78** [comment]
Wow that was quick.

I'm using version Winx64_4.0.0 of Tezbox. My address is :

KT1E5VcMayvVCmXutrcpNH9JArsaPeAkF6jC

No matter the fees I try, the defaults or 0, I get the "Uh-Oh! Operation Failed! Please check your inputs

I've tried sending to my Main tz1 address within the Tezbox wallet as well as to Bakechain.

Sent from ProtonMail mobile

-------- Original Message --------

**Comment by stephenandrews** [comment]
Can you please let me know the exact details of your transaction, the to address, amount, fees, and other settings please

**Comment by sbusher78** [comment]
Trying to send 151 XTZ from the address I gave you to tz1bUcjyYvGr4Sr433wnedh9ZF8t93hWyLh4

I used Medium-5000ųtz as well as Low-1420ųtz for the fees. I also tried to send a single xtz with the same fee combos to and from the same addresses

Sent from ProtonMail mobile

-------- Original Message --------

**Comment by stephenandrews** [comment]
Can you confirm the advanced settings for me too please. Also, on the bottom left where it shows your connection can you tell me what it says down there (the RPC address you are connected to as well as the status and the block number)

**Comment by sbusher78** [comment]
Sure the advanced options are as follows:

Parameters is blank, Gas Limit is at the default of 10300, and the Storage limit is at the default of 300.

Lower left says:

NetxdQprcVkpaWU
Level 387889

"Connected to Mainnet_003
Https://mainnet.tezrpc.me

Sent from ProtonMail mobile

-------- Original Message --------

**Comment by stephenandrews** [comment]
That all looks fine - can't see any issues. Make sure you don't have any additional characters in the address or the amount field, and make sure you don't have any firewalls etc blocking the outgoing txs. I'm not having issues with the same release and I don't see any issues with your settings so far

**Comment by sbusher78** [comment]
Yeah I checked all that and it's all fine. So weird man. I have quite a bit of tezos in the wallet and I'm a bit nervous. Do you think I can restore the wallet in Bakechain using the private keys? I know last time when I had an excess 300 XTZ in BakeChain i was able to restore those coins in TezBox so wasn't sure if the reverse was possible.

Sent from ProtonMail mobile

-------- Original Message --------

**Comment by sbusher78** [comment]
I've now restored the wallet in the Android version of TezBox and still cant send. I dont get an error from the app it says it's been sent but it never arrives in the destination and my balance is never debited. Am I hosed? I have over 10,000 XTZ in the wallet.

Sent from ProtonMail mobile

-------- Original Message --------

**Comment by sbusher78** [comment]
So you're not going to believe this but I just tried again, changing nothing, and it worked!! Very weird but im just happy it works. As always thanks so much for your assistance you're an asset to the Tezos community!

Sent from ProtonMail mobile

-------- Original Message --------

**Comment by sbusher78** [comment]
Hi Stephen,

Off subject but do I have to keep extra XTZ in Bakechain to cover the bond or can I have exactly 10,000? I thought I read you should have an extra 15% or so in there so I have 10,150 but if that's not needed I'll add that 150 to thr amount I have in my Tezbox wallet which is delegating to a Baker. Thanks in advance!

Sent from ProtonMail mobile

-------- Original Message --------
